sensor/kubernetes/fake/flows.go

```diff
@@ -243,8 +253,8 @@ func (w *WorkloadManager) getFakeNetworkConnectionInfo(workload NetworkWorkload)
 		conns = append(conns, conn)
 		if endpointPool.Size < endpointPool.Capacity {
 			endpointPool.add(networkEndpoint)
-			networkEndpoints = append(networkEndpoints, networkEndpoint)
 		}
+		networkEndpoints = append(networkEndpoints, networkEndpoint)
```

Comment:
Now listening endpoints will be sent even if the endpointPool is at capacity. This means that an unlimited number of listening endpoints can be sent and in the central database. The endpointPool is still kept so that listening endpoints will be closed, so that pathway is still tested.